bitnami/apisix/crds/customresourcedefinitions.yaml

```diff
@@ -0,0 +1,3041 @@
+#
```

Comment:
Is there any reason to no include 'ApisixGlobalRule.yaml' here?

In addition, I would like to make the same request here, please add an upstream link to help tracking outdated CRDs.

Reply:
Just did the same as upstream